MultiplayerMod/ModRuntime/Context/ExecutionLevel.cs

```diff
@@ -1,9 +1,9 @@
 ﻿namespace MultiplayerMod.ModRuntime.Context;
 
 public enum ExecutionLevel {
```

Comment:
Level names are still quite confusing for me.
Could we at least  copy the comments here from your description?
As another idea: perhaps instead of using component names (which are subject to change) use description as name?
E.g. Always, OnlyInMultiplayerGame, AfterInitilization, OnlyInCommand, ??.

Reply:
Sure, I'll add the comments. But I'm quite unsure about using description as names (maybe because I see this as a different domain). And the example names don't look like level names to me, I mean, yeah these are just examples, but anyway it's hard to came up with correct level naming in this format. Maybe `Runtime` can be called `Gameplay` tho since it's used for all interactions yeah.